code/modules/mod/modules/modules_maint.dm

```diff
@@ -48,12 +48,60 @@
 		return
 	mod.wearer.visible_message("[src] inside [mod.wearer]'s [mod.name] snaps shut, mutilating the user inside!", span_userdanger("*SNAP*"))
 	mod.wearer.emote("scream")
+	mod.wearer.Paralyze(10000, TRUE)
```

Comment:
are we sure during revival this doesnt keep going

Reply:
TRUE doesnt tell someone reading the code anything, use the named parameter

Reply:
Alright